server/instances.go

```diff
@@ -151,6 +151,29 @@ func (p *Plugin) InstallInstance(instance Instance) error {
 				}
 			}
 
+			if instance.Common().Type == CloudOAuthInstanceType && len(instances.IDs()) > 0 && instances.checkIfExists(instance.GetID()) {
+				p.API.LogDebug("Getting the stored JWT instance data to store it inside the OAuth instance data.")
+				jwtInstance, err := p.instanceStore.LoadInstance(instance.GetID())
```

Comment:
```suggestion
				previousInstance, err := p.instanceStore.LoadInstance(instance.GetID())
```

We should probably also rename the function parameter `instance` to `newInstance` as well